Documentation/deploy-master.md

```diff
@@ -668,6 +668,14 @@ kube-controller-$node
 kube-proxy-$node
 ```
 
+### Configure Calico
+
+To start Calico services :
+
+```
+docker run --rm --net=host -v /srv/kubernetes/manifests:/host/manifests quay.io/coreos/hyperkube:v1.5.1_coreos.0 /hyperkube kubectl apply -f /host/manifests/calico.yaml
```

Comment:
I'm actually a little bit hesitant to go with either of the above approaches.

It's really important that if a user wants to install Calico, that Calico is installed before other addons.  

For example, if the DNS add-on is installed first, then it won't get networked with Calico and thus won't have Calico policy applied to it.  Ideally it is done as part of installing the master, or immediately after installing the nodes and before installing any other add-on.

In the "Calico only" case, the add-on manager would be fine since no other pods could start until Calico was installed, but since users are also installing Flannel, it will now be possible for pods to start prior to Calico getting installed, which would be a problem.